app/src/main/java/com/example/b07group7project/StoreOwnerActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.example.b07group7project;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+
+import com.example.b07group7project.Order;
+import com.example.b07group7project.OrderAdapter;
+import com.example.b07group7project.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StoreOwnerActivity extends AppCompatActivity {
+    private List<Order> pendingOrdersList = new ArrayList<>();
+
+    // Initialize RecyclerView, Adapter, and LayoutManager
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_store_owner);
+
+        // Assuming you have RecyclerView, Adapter, and LayoutManager set up.
+        RecyclerView recyclerView = findViewById(R.id.recyclerView);
+        LinearLayoutManager layoutManager = new LinearLayoutManager(this);
+        recyclerView.setLayoutManager(layoutManager);
+
+        // Fetch pending orders from the database or your data source.
+        // For the sake of this example, we'll create dummy data.
+        pendingOrdersList = getPendingOrders();
```

Comment:
redo this part so it works async. Basically instead of returning a List<Order>, you should provide a function that will be called with the list of orders

Reply:
https://github.com/sjingoi/B07Group7Project/pull/17/commits shows an example refactoring to get that to work